pagopa/io-app
# fix: [IA-881] Wrong message for Outcome code 4

## Short description
This pr changes the copy and the icon for the `outcome code 4` 

Before            |  After
:-------------------------:|:-------------------------:
<img src="https://user-images.githubusercontent.com/26501317/172667638-88b1ca21-3b34-402b-802c-9e2cf8636a4c.png" width="300">|  <img src="https://user-images.githubusercontent.com/26501317/172667690-05939701-981d-4abf-a18e-a8f4be772719.png" width="300">

## List of changes proposed in this pull request
- Copy update
- Image update

## How to test
- On dev server, change https://github.com/pagopa/io-dev-api-server/blob/master/src/routers/payment.ts#L214 with return code `4`
- Try a payment and verify the new copy / image.
